source/overview/faq.rst

```diff
@@ -507,7 +507,7 @@ The trade-off with this design is that if physical security is not in effect, an
 
 For organizations who choose to deploy in such a configuration, please consider the following mitigations:
 
-  1. Instead of username-password, use a single-sign-on provider in Mattermost Enterprise Edition like OneLogin, Okta, or ADFS, or use the open source GitLab SSO option available with Mattermost Team Edition.
+  1. Instead of username-password, use a Single-sign-on (SSO) provider in Mattermost Enterprise Edition like OneLogin, Okta, or ADFS, or use the open source GitLab SSO option available with Mattermost Team Edition.
```

Comment:
```suggestion
  1. Instead of username-password, use a Single Sign-On (SSO) provider in Mattermost Enterprise Edition like OneLogin, Okta, or ADFS, or use the open source GitLab SSO option available with Mattermost Team Edition.
```